java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -509,7 +509,8 @@ public static ColumnVector stringConcatenate(ColumnView[] columns) {
    * @return A new java column vector containing the concatenated strings.
```

Comment:
Need to update the comment here to match the behavior.

Reply:
Fixed! Thanks for catching that.